apps/okreads/browser/src/app/app.component.html

```diff
@@ -6,7 +6,8 @@ <h1>okreads</h1>
         <div>
           <button
             data-testing="toggle-reading-list"
-            mat-flat-button class="reading-list"
+            mat-button class="reading-list"
+            aria-label="reading-list-button"
```

Comment:
Aria labels should be _readable_ by a screen reader. i.e. plain english, there's no need for underscores.